android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.java

```diff
@@ -33,6 +33,28 @@ public static void setBundleUrl(final String bundleUrl) {
     sBundleUrl = bundleUrl;
   }
 
+  @DoNotStrip
+  public static void reloadFromManifest(final int activityId) {
+    try {
+      Class.forName("host.exp.exponent.kernel.Kernel")
```

Comment:
i'm confused about why the methods in this class are all using reflection to call static methods on Kernel (which is in the same package...) but maybe best to just leave it alone at this point 🤷‍♂

Reply:
yeah i'm not sure either but i just did what other things in here were doing